design.md

```diff
@@ -1,13 +1,14 @@
 # Design
 
 ## Who is the target for this design, e.g., are you assuming any knowledge on the part of the language users?
-
+The target for this design is someone who understands how to call functions and use conditionals in a programming language.
 ## Why did you choose this design, i.e., why did you think it would be a good idea for users to express the maze-searching computation using this syntax?
-
+I chose this design because it would allow students who have gone through CS5, to apply the skills they have learned to more easily perform the tasks required by the picobot assignment which were previously harder to do.
 ## What behaviors are easier to express in your design than in Picobot’s original design?  If there are no such behaviors, why not?
-
+It is easier to express if a direction is 'open' or 'blocked' than in Picobot's original design.
 ## What behaviors are more difficult to express in your design than in Picobot’s original design? If there are no such behaviors, why not?
-
+It will be more difficult to express which direction you are moving towards and the new state in a succinct manner. This is because I plan on using calls to move() which will make it require more code to express the change to picobot. I think this change is beneficial because it will make it more explicit what changes are being made to picobot's state.
```

Comment:
I think this is an interesting point - that the code might look more complicated, but can express more information explicitly.